cypress/support/index.ts

```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
```

Comment:
looks like these came with packae. safe to remove?

Reply:
Yes, it's safe to remove these files. They are needed when the application begins to get complex, though.